app/src/components/features/rules/RulePairs/Pairs/Rows/RowsMarkup/RequestBodyRow/index.js

```diff
@@ -22,6 +23,10 @@ const RequestBodyRow = ({
     GLOBAL_CONSTANTS.REQUEST_BODY_TYPES.STATIC
   );
   const [isCodeFormatted, setIsCodeFormatted] = useState(false);
+  const [isCodeMinified, setIsCodeMinified] = useState(false);
```

Comment:
`isCodeMinified` should be initialized with `true`, as we are always showing the minified JSON in code editor by default.

also set this to `true` when switching from programmatic mode to static mode.